dist/rust/riot-rust-linker-hook

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
```

Comment:
At least you can be nice to non-linux people and change to `#!/usr/bin/env bash` as requested ;-)